### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by BridgeAR**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,6 +32,9 @@ doctor.collect(['node', './path-to-script.js'], function (err, filepath) {
 })
 ```
 
+You can find examples in
+[node-clinic-doctor-examples](https://github.com/nearform/node-clinic-doctor-examples/blob/master/slow-io/index.js#L29)
```
Comment (mcollina) [inline README.md]: Shouldn't this be pointing to the full repo?

Comment (BridgeAR) [reply README.md]: Yes, likely. I'll change that as well.

Comment (BridgeAR) [reply README.md]: Addressed

**Comment by BridgeAR** [comment]
Rebased.

**Pull request merged**